fail to pull mondrian dependency

`mvn install` fails with the following messages.

```
[ERROR] Failed to execute goal on project h2: Could not resolve dependencies for project com.github.vlsi.calcite:h2:pom:1.0-SNAPSHOT: Failure to find pentaho:mondrian:jar:4.2.0.0-190 in http://ivy-nexus.pentaho.org/content/groups/omni was cached in the local repository, resolution will not be reattempted until the update interval of pentaho-repo has elapsed or updates are forced -> [Help 1]
```

It seems mondrian 4.2.0.0-190 is not available on pentaho repository anymore, I can't find it on http://nexus.pentaho.org/content/groups/omni/pentaho/mondrian/

--- a/foodmart-loader/pom.xml
+++ b/foodmart-loader/pom.xml
@@ -36,7 +36,7 @@
     </dependency>
 
     <dependency>
-      <groupId>pentaho</groupId>
+      <groupId>org.pentaho</groupId>
       <artifactId>mondrian</artifactId>
     </dependency>
   </dependencies>

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -64,9 +64,9 @@
       </dependency>
 
       <dependency>
-        <groupId>pentaho</groupId>
+        <groupId>org.pentaho</groupId>
         <artifactId>mondrian</artifactId>
-        <version>4.2.0.0-190</version>
+        <version>4.5.0.7-258</version>
       </dependency>
       <dependency>
         <groupId>net.hydromatic</groupId>

